app/models/member.rb

```diff
@@ -239,9 +239,8 @@ def update_newsletter_subscription
     end
   end
 
-  def newsletter_subscribe(testing = false)
+  def newsletter_subscribe(gb = Gibbon::API.new, testing = false)
```

Comment:
@Br3nda: I changed the method so I could inject a double :-) I looked at using `any_instance`, but saw it was deprecated; OTOH, I ended up using `receive_message_chain`, which is also deprecated. Should I just go ahead and use `any_instance`, or can you suggest a better way?